web-ui/src/components/reviews/periods/ReviewPeriodStepper.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import Box from '@mui/material/Box';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import optionsArr from './reviewStatus.json';
```

Comment:
Is this used in more than one place?

Reply:
Not yet, but it may be eventually. Wanted to keep it separate for visibility.